src/main/kotlin/pl/edu/agh/gem/annotation/nullorpattern/NullOrPattern.kt

```diff
@@ -0,0 +1,15 @@
+package pl.edu.agh.gem.annotation.nullorpattern
+
+import jakarta.validation.Constraint
+import kotlin.reflect.KClass
+
+@Target(AnnotationTarget.FIELD)
+@Retention(AnnotationRetention.RUNTIME)
+@MustBeDocumented
```

Comment:
why we add that annotation ?

Reply:
should i remove it?

Reply:
(NullOrPattern works without this annotation)

Reply:
if we dont use that, and that lib is not public api, IMHO better to remove